templates/control/edit_commissionprices.html

```diff
@@ -164,6 +161,43 @@
 
       <button type="submit" class="button positive" style="float: right;">Save Description</button>
     </form>
+
+    <div class="di-tags">
+      <h3>Preference Tags <a class="modify typeface-default color-c" href="#">Modify</a></h3>
+      <p>
+        Use these tags to indicate subject matter you prefer to draw, making it easier for
+        commissioners to find you!
+      </p>
+
+      <div class="tags open clear">
+        $if query['tags']:
+          $for tag in query['tags']:
+            <a href="/marketplace?q=${tag}">${tag}</a>
+        $else:
+          <p class="color-lighter">You have not defined any Artist Tags!</p>
+      </div>
+
+      <h3 class="edit">Edit Tags</h3>
+      $:{RENDER("common/detail_tag_form.html", [query['userid'], "user", query['tags']])}
+
+    </div><!-- /di-tags -->
+    <div class="di-tags">
+      <h3>Will-Not-Draw Tags <a class="modify typeface-default color-c" href="#">Modify</a></h3>
```

Comment:
Perhaps, but we certainly can't call them blacklist tags again :b
Maybe "opt-out"? "avoidance tags"? "the no-no list"?